app/node/generator/text/textv3.cpp

```diff
@@ -150,11 +150,12 @@ void TextGeneratorV3::GenerateFrame(FramePtr frame, const GenerateJob& job) cons
 
   // Draw rich text onto image
   QPainter p(&img);
-  p.scale(1.0 / frame->video_params().divider(), 1.0 / frame->video_params().divider());
+  const double par = frame->video_params().pixel_aspect_ratio().toDouble();
```

Comment:
Long explicite names are prefered so something like `pixel_aspect_ratio` would be better